spec/models/manageiq/providers/openstack/cloud_manager/provision/volume_attachment_spec.rb

```diff
@@ -5,6 +5,8 @@
     @flavor = FactoryBot.create(:flavor_openstack)
     @volume = FactoryBot.create(:cloud_volume_openstack)
 
+    # We're storing objects in the instance_type, so we must permit loading this class
+    ActiveRecord::Base.yaml_column_permitted_classes = ActiveRecord::Base.yaml_column_permitted_classes | [@flavor.class]
```

Comment:
I'm not a big fan of this but if we use YAML for serialized columns and want to use safe_load, we need to explicitly allow classes to be YAML.loaded at the point we deserialize them... or we have to populate this with all possible models.

Reply:
Agreed AR instances in MiqRequestTask options is really terrible, but it is everywhere